modin/pandas/dataframe.py

```diff
@@ -1161,7 +1161,11 @@ def insert(self, loc, column, value, allow_duplicates=False):  # noqa: PR01, D20
                 and not isinstance(value, (pandas.Series, Series))
                 and len(value) != len(self.index)
             ):
-                raise ValueError("Length of values does not match length of index")
+                raise ValueError(
+                    "Length of values ({}) does not match length of index ({})".format(
+                        len(value), len(self.index)
+                    )
+                )
```

Comment:
In the process of manual testing, I accidentally found a message mismatch in an exception regarding pandas.